int32 movement_type		# 0 = standard. 1 = random. 2 = ...etc.
---
bool success